crates/uv-cli/src/lib.rs

```diff
@@ -549,6 +554,10 @@ pub struct VersionArgs {
     #[arg(long)]
     pub short: bool,
 
+    /// Permit the result of `--bump` to be a version decrease
+    #[arg(long)]
+    pub allow_decreases: bool,
```

Comment:
Is there a use case for this flag? If so, it might be nice to mention it in the docs here. I'm kind of having trouble envisioning one... If there isn't a use case or if it's really obscure, would it make sense to jettison this extra option for now?